src/connector/TypeScript-REST/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "td2.0-rest-connector",
-  "version": "1.0.0",
+  "version": "1.0.3",
```

Comment:
why bump to 1.0.3?

Reply:
version < 1.0.3  has critical bug. Fix those bug in current version.